src/main/java/cubicchunks/world/cube/Cube.java

```diff
@@ -786,4 +791,8 @@ public boolean hasLightUpdates() {
         LightingManager.CubeLightUpdateInfo info = this.getCubeLightUpdateInfo();
         return info != null && info.hasUpdates();
     }
+
+    public void markEdgeNeedSkyLightUpdate(EnumFacing side) {
+        this.edgeNeedSkyLightUpdate[side.ordinal()] = true;
```

Comment:
This is not saved to disk. This is bad. Look into cube saving code. Also, the default values when the flags don't exist should be "true" so that it will try to at least partially try to fix lighting in existing worlds.